account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -0,0 +1,150 @@
+import {
+    custom,
+    keccak256,
+    parseEther,
+    publicActions,
+    toHex,
+    type Address,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Address' is defined but never used.
<!-- __reviewdog__:ChAyYTdhNDRiNDk3MDU2OGZlEgZyZGpzb24= -->